lib/src/dialogs/macos_alert_dialog.dart

```diff
@@ -89,106 +116,262 @@ class MacosAlertDialog extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final brightness = MacosTheme.brightnessOf(context);
+
+    final outerBorderColor = brightness.resolve(
+      Colors.black.withOpacity(0.23),
+      Colors.black.withOpacity(0.76),
+    );
+
+    final innerBorderColor = brightness.resolve(
+      Colors.white.withOpacity(0.45),
+      Colors.white.withOpacity(0.15),
+    );
+
     return Dialog(
-      backgroundColor: MacosTheme.brightnessOf(context).isDark
-          ? MacosColors.controlBackgroundColor.darkColor
-          : MacosColors.controlBackgroundColor,
+      backgroundColor: brightness.resolve(
+        CupertinoColors.systemGrey6.color,
+        MacosColors.controlBackgroundColor.darkColor,
+      ),
       shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(7.0),
+        borderRadius: _kDialogBorderRadius,
       ),
-      child: ConstrainedBox(
-        constraints: BoxConstraints(maxWidth: 260),
-        child: Column(
-          mainAxisSize: MainAxisSize.min,
-          children: [
-            const SizedBox(height: 28),
-            ConstrainedBox(
-              constraints: BoxConstraints(
-                maxHeight: 56,
-                maxWidth: 56,
+      child: Container(
```

Comment:
Would it be possible to use a DecoratedBox rather than a Container here?

Reply:
I considered it, but DecoratedBox has no foregroundDecoration, meaning that I'd have to use two boxes to achieve the same. And I need the padding property of the container too.